package.json

```diff
@@ -19,7 +19,9 @@
     "test": "ts-node src/run_tests.ts",
     "coverage": "nyc ts-node src/run_tests.ts",
     "link-local": "yalc link",
-    "publish-local": "yarn prep && yarn build && yalc push"
+    "publish-local": "yarn prep && yarn build && yalc push",
```

Comment:
NIT: (not your fault) can we alpha-order these?

Reply:
Done.